**Due**:Tuesday, June 3, 2021 11:59 PM EST

**Author**:Alejandro M. Ouslan

# Exercice
- Two soccer (Football) teams go into a penalty shoot-out. Each team selects their five
players for the shootout. The probabilities of them making their individual, independent
shot is as follows:
    - Team A: {0.89, 0.67, 0.78, 0.44, 0.75}
    - Team B: {0.91, 0.75, 0.39, 0.60, 0.88}

In [1]:
# imports 
import polars as pl
import pandas as pd
import numpy as np
from scipy.stats import norm, binom, t, ttest_1samp, ttest_ind

In [2]:
team_a = np.array([0.89, 0.67, 0.78, 0.44, 0.75])
team_b = np.array([0.91, 0.75, 0.39, 0.60, 0.88])

## Exercice a:
- What is the probability of each team’s five players scoring their penalty? Which team
has the higher probability?

In [3]:
num = 1
for i in team_a:
    num = num * i
print(num)

0.15348762000000002


In [4]:
num = 1
for i in team_b:
    num = num * i
print(num)

0.14054039999999998


- **Answer**: Team A has the higher probability.

## Exercice b:
- Suppose the second player in Team A misses, and so does the third player of Team B; what is the probability of the remaining four players in each team scoring their penalty?


In [5]:
miss_a = np.delete(team_a, 1)
miss_b = np.delete(team_b, 2)
print(miss_a.mean())
print(miss_b.mean())

0.715
0.785


- **Answer**: Team A now has a probability of $0.715$ and Team B now has a probability of $0.785$

## Exercice c:
- Now, suppose the second player of each team has a conditional probability based on whether the first penalty taker scores their penalty (assume the probability of the first penalty taker stays the same):
  - For Player 2 in Team A:
  $$f_{1|2}(1|1) = 0.83,f_{1|2}(0|1) = 0.17$$
  $$f_{1|2}(1|0) = 0.60,f_{1|2}(0|0) = 0.40$$

  - For Player 2 in Team B:
  $$f_{1|2}(1|1) = 0.81,f_{1|2}(0|1) = 0.19$$
  $$f_{1|2}(1|0) = 0.72,f_{1|2}(0|0) = 0.28$$

- Assuming the remaining players are independent of players one and two, what is the
probability of the whole team if:

## Exercice d:
  - The first shooter from both teams scores. Which team has the higher probability?

In [6]:
hit_a = np.delete(team_a, 0)
hit_b = np.delete(team_b, 0)

hit_a[hit_a == 0.67] = 0.83
hit_b[hit_b == 0.75] = 0.81

print(hit_a.mean())
print(hit_b.mean())

0.7
0.67


- **Answer**: Team A has the higher probability with 0.7

## Exercise e:

- Team A’s first player scores, but the second player does not.

In [7]:
miss_a = np.delete(team_a, 0)
miss_a = np.delete(team_a, 0)

print(miss_a.mean())

0.66


**Answer**: The probability of the remading players will be 0.66

# Excercise
Let X be a random variable distributed as Normal (6,3). Find the probabilities of the following events:

1. $P(X <= 6)$
2. $P(X > 4)$
3. $P(|X - 5| > 1)$

In [8]:
mean, std = 6, 3
print(norm(loc=mean, scale=std).cdf(6))
print(1 - norm(loc=mean, scale=std).cdf(4))


0.5
0.7475074624530771


In [9]:
upper = norm(loc=mean, scale=std).cdf(6)
lower = norm(loc=mean, scale=std).cdf(4)

print(1 - (upper - lower))

0.7524925375469229


**Answers**:

1. $P(X <= 6) = 0.5$
2. $P(X > 4) = 0.7475$
3. $P(|X - 5| > 1) = 0.7525$

# Excercise 

- Just prior to jury selection for O. J. Simpson’s murder trial in 1995, a poll found that about 20% of the adult population believed Simpson was innocent (after much of the physical evidence in the case had been revealed to the public). Ignore the fact that this 20% is an estimate based on a subsample from the population; for illustration, take it as the true percentage of people who thought Simpson was innocent prior to jury selection Assume that the 12 jurors were selected randomly and independently from the population (although this turned out not to be true).

## Excercie i
- Find the probability that the jury had at least one member who believed in Simpson’s innocence prior to jury selection. [Hint: Define the Binomial ($12,20)$ random variable $X$ to be the number of jurors believing in Simpson’s innocence].

In [10]:
# Calculating the Binomial CDF for n >= 1
k = 1
n = 12
p = .2
1 - binom.cdf(k, n, p)  + binom.pmf(k, n, p)

0.931280523264

**Answer**: The probability that at least 1 jury found Simpsone innocent is $0.9313$

## Excerce ii
- Find the probability that the jury had at least two members who believed in Simpson’s innocence. [Hint: $𝑃(𝑋 ≥ 2) = 1 − 𝑃(𝑋 ≤ 1)$ and $𝑃(𝑋 ≤ 1) = 𝑃(𝑋 = 0) + 𝑃(𝑋 = 1)$.]

In [11]:
# Calculating the Binomial CDF for n >= 2
k = 2
n = 12
p = .2
1 - binom.cdf(k, n, p)  + binom.pmf(k, n, p)

0.7251220930560002

**Answer**: The probability that at least 1 jury found Simpsone innocent is $0.9313$

# Excercice
Let X denote the annual salary of university professors in the United States, measured in
thousands of dollars. Suppose that the average salary is 52.3, with a standard deviation of
14.6. Find the mean and standard deviation when salary is measured in dollars.

In [12]:
# transforming from thousans of dollars to dollars 
print(52.3*1000)
print(14.6*1000)

52300.0
14600.0


**Answer**: The mean salary is $52300$ and the standard deviation is $14600$

# Excercice
- Let $Y_1,Y_2,Y_3,$ and $Y_4$ be independent, identically distributed random variable from a ppopulation with mean $\mu$ and variene $\sigma^2$. Let $\overline Y = \frac{1}{4}(Y_1+Y_2+Y_3+Y_4)$ denote the average of these four random variables.

## Excercice i
- What are the expecteed values and varience of $\overline Y$ in terms of $\mu$ and $\sigma^2$?

In [13]:
%%latex
\begin{align}
    E[\overline Y] &= E[\frac{1}{4}(Y_1+Y_2+Y_3+Y_4)] \\
    E[\overline Y] &= \frac{1}{4}(E[Y_1]+E[Y_2]+E[Y_3]+E[Y_4]) \\
    E[\overline Y] &= \frac{1}{4}(\mu_1+\mu_2+\mu_3+\mu_4) \\
    E[\overline Y] &= \mu \\
    \end{align}
    
\begin{align}
    \sigma^2 &= \frac{1}{n} \sum_{i=1}^4 (Y_i - \mu) \\
    \end{align}

<IPython.core.display.Latex object>

## Excercice ii

- Now, consider a different estimator of 𝜇:
        $$W = \frac{1}{8} Y_1 + \frac{1}{8} Y_2 + \frac{1}{4} Y_3 + \frac{1}{2} Y_4$$

In [14]:
%%latex
\begin{align}
    E[W] &= E[\frac{1}{8} Y_1 + \frac{1}{8} Y_2 + \frac{1}{4} Y_3 + \frac{1}{2} Y_4] \\
    E[W] &= \frac{1}{8} E[Y_1] + \frac{1}{8} E[Y_2] + \frac{1}{4} E[Y_3] + \frac{1}{2} E[Y_4] \\
    E[W] &= \frac{1}{8} \mu_1 + \frac{1}{8} \mu_2 + \frac{1}{4} \mu_2 + \frac{1}{2} \mu_3 \\
    E[W] &= \mu
    \end{align}
    
\begin{align}
    \sigma^2 &= \frac{1}{n} \sum_{i=1}^4 (Y_i - \mu) \\
    \end{align}

<IPython.core.display.Latex object>

## Excercice iii
- Based on your answers to parts i and ii, which estimator of m do you prefer,
$\overline Y$ or $W$?
    - I would prefer $\overline Y$ due to its simplicity however $W$ would be necesary depending on the problem.

# Excercice 
- Suppose that between their first and second years in college, 400 students are randomly
    selected and given a university grant to purchase a new computer. For student i, 𝑦𝑖
    denotes the change in GPA from the first year to the second year. If the average change is $\overline Y = .132$ with standard deviation $s = 1.27$, is the average change in GPAs statistically greater than zero?
    
    - Doing a t-test, we find that the p-value is $0.98$, so there is not enough evidence to reject the null hypothesis.

# Excercice 
The new management at a bakery claims that workers are now more productive and they
were under old management, which is why wages have “generally increased.” Let $𝑊_𝑖^𝑏$ be
worker i’s wage under the old management and let $𝑊_𝑖^𝑎$ be Worker i’s wage after the
change. The difference is $𝐷_𝑖 \equiv 𝑊_𝑖^𝑎 − 𝑊_𝑖^𝑏$. Assume that the $D_i$ are a random sample from a Normal $(\mu, \sigma^2)$ distribution.

## Excercice i
- Using the following data on 15 workers, construct an exact 95% confidence
interval for 𝜇.

In [15]:
df = pl.DataFrame(
    {
        "before": [8.30, 9.4, 9, 10.5, 11.4, 8.75, 10, 9.5, 10.8, 12.55, 12, 8.65, 7.75, 11.25, 13.65],
        "after": [9.25, 9, 9.25, 10, 12, 9.5, 10.25, 9.5, 11.5, 13.1, 11.5, 9, 7.75, 11.5, 13]
    }
)
t.interval(confidence=.95, df=14, loc=df.mean(), scale=df.std())

(array([[6.59132867, 6.98510843]]), array([[13.87533799, 13.8282249 ]]))

**Answer**: CI for before wage increase is $[6.59, 13.88]$ and after wage increase is $[6.99, 13.83]$.

## Excercice ii
- Formally state the null hypothesis that there has been no change in average wages. In particular, what is the 𝐸(𝐷𝑖) under 𝐻0? If you are hired to examine the validy of the new management’s claim, what is the relevant alternative hypothesis in terms of 𝜇 = 𝐸(𝐷𝑖)?
$$ \text{Null hypothesis: } H_0: \mu_{a} = \mu_{b} \\

\text{Alternative hypothesis: } H_1: \mu_a > \mu_b $$


## Excercise iii
- Test the null hypothesis from part ii against the stated alternative at the 5% and 1% levels.

In [16]:
t_statistic, p_value = ttest_ind(df["after"], df["before"], alternative="greater")
# Output the results
print(f"t-statistic: {t_statistic}")
print(f"P-value: {p_value}")

t-statistic: 0.2881320751359442
P-value: 0.38768388434956697


**Answeer**: Given that the P-value is $0.39$ we can not reject the null hypothesis for either $\%1$ or $\%5$.